File: Modul-5/Soal 4/PRAK504-2410817120019-NadyaRamadhani.py
def reverse(nil):
    reversed = 0
    while nil > 0:
        reversed = reversed * 10 + (nil % 10)
        nil = nil // 10
    return reversed

A, B = map(int, input().split())
A=reverse(A)
B=reverse(B)
C = A+B
print(reverse(C))

File: Modul-5/Soal 4/PRAK504-2410817120019-NadyaRamadhani.c
#include <stdio.h> 
int reverse(int nil){
    int reversed = 0;
    while(nil>0){
        reversed = reversed * 10 + nil % 10;
        nil = nil / 10;
    }
    return reversed;
}

int main() {
    int A, B;
    scanf("%d %d",&A,&B);
    A=reverse(A);
    B=reverse(B);
    int C = A+B;
    printf("%d",reverse(C));

    return 0;
}